examples/lib/stories/experimental/layout_component_example.dart

```diff
@@ -0,0 +1,107 @@
+import 'dart:async';
+
+import 'package:flame/components.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flutter/material.dart';
+
+class LayoutComponentExample extends FlameGame {
+  static const String description = '''
+    On this example, click on the "Skip" button to display all the text at once.
```

Comment:
This description doesn't match

Reply:
Oh my god I completely forgot about this